src/components/picker/index.tsx

```diff
@@ -208,10 +194,44 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
         onCancel: cancelSelect
       };
 
-      return renderCustomModal(modalProps);
+      return renderOverlay(modalProps);
     }
   };
 
+  const renderTextField = useCallback(() => {
+    return renderInput ? (
+      // @ts-expect-error - hopefully will be solved after the picker migration ends
+      renderInput(value, label)
+    ) : (
+      <TextField
+        // @ts-expect-error
+        ref={pickerRef}
+        {...others}
```

Comment:
others now includes all the props we are passing to the newProps hook. Let's make sure there is no prop that can affect the TextField here

Reply:
Verified, it's ok